src/Pos/Context/Functions.hs

```diff
@@ -29,25 +30,36 @@ import           Data.Time           (diffUTCTime, getCurrentTime)
 import           Data.Time.Units     (Microsecond, fromMicroseconds)
 import           Ether.Internal      (HasLens (..))
 
-import           Pos.Context.Context (BlkSemaphore (..), GenesisStakes (..),
-                                      GenesisUtxo (..), StartTime (..))
-import           Pos.Core            (HeaderHash, SlotLeaders, StakesMap)
+import           Pos.Context.Context (BlkSemaphore (..), GenesisUtxo (..), StartTime (..))
+import           Pos.Core            (HeaderHash, SlotLeaders, StakeholderId, StakesMap)
 import           Pos.Genesis         (genesisLeaders)
 import           Pos.Lrc.Context     (lrcActionOnEpoch, lrcActionOnEpochReason, waitLrc)
-import           Pos.Txp.Toil.Types  (Utxo)
+import           Pos.Txp.Toil        (Utxo, utxoToStakes)
+import           Pos.Util.Util       (getKeys)
 
 ----------------------------------------------------------------------------
 -- Genesis
 ----------------------------------------------------------------------------
 
-genesisUtxoM :: (Functor m, MonadReader ctx m, HasLens GenesisUtxo ctx GenesisUtxo) => m Utxo
+genesisUtxoM ::
+       (Functor m, MonadReader ctx m, HasLens GenesisUtxo ctx GenesisUtxo)
```

Comment:
Similarly, don't reformat without functional changes. The signature should be `(MonadReader ctx m, HasGenesisUtxo ctx) => m Utxo`.